goon/browserassets/js/browserOutput.js

```diff
@@ -72,7 +72,11 @@ var opts = {
 	'enableEmoji': true,
 
 	// Reboot message stuff
-	'rebootIntervalHandler': null
+	'rebootIntervalHandler': null,
+
+	// Syndicate codewords
+	'codePhrases': [],
```

Comment:
If someone crashes and re-opens their game, do they loose the chat highlighting of the codewords?

And if it is persistent, when are these entries cleared out?

Reply:
I have overlooked these issues, which are now fixed.